specifyweb/frontend/js_src/lib/components/Forms/SubView.tsx

```diff
@@ -269,7 +266,16 @@ export function SubView({
             }
           />
         </ReadOnlyContext.Provider>
-      ) : undefined}
+      ) : isButton ? undefined : (
+        <DataEntry.SubForm>
+          <DataEntry.SubFormHeader>
+            <DataEntry.SubFormTitle>
+              {relationship.label}
+            </DataEntry.SubFormTitle>
+          </DataEntry.SubFormHeader>
+          {commonText.loading()}
+        </DataEntry.SubForm>
```

Comment:
I'd like to move the code for this to [RecordSelectorFromCollection](https://github.com/specify/specify7/blob/production/specifyweb/frontend/js_src/lib/components/FormSliders/RecordSelectorFromCollection.tsx) rather than having it so high up in the prop tree in Subview, but that requires a little more refactoring which also somewhat interferes with #5051. 

Definitely doable, but maybe not for the `7.9.7` release.